### ISSUE TITLE
Some plugins are not exposed from package

### ISSUE THREAD

**Issue by alvar-osz** [graphql-hive/envelop#96]
**Describe the bug**

Some plugins, such as for example `useExtendContext`, are mentioned in the documentation but not exposed as part of the package.

**To Reproduce**

Write

```
import { envelop, useSchema, useExtendContext } from '@envelop/core';
```

and get `Error: Cannot find module '@envelop/core/plugins/use-extend-context'`

I expect to be able to get the required plugin imported to my file.

**Environment:**

- OS: macOS 11.2.3
- `@envelop/core`:
- NodeJS: v15.13.0

**Comment by dotansimha** [comment]
Hi @ajotaos , sorry for the inconvenience!

Fixed in https://github.com/dotansimha/envelop/commit/3b9277961f94eeff0760dacf7ce20c07e183e085 , available as `@envelop/core@0.1.1`.

**Comment by alvar-osz** [comment]
Oh my god, fastest fix ever. Thank you @dotansimha 🤩
